MAINT: Remove unnecessary 'from __future__ import ...' statements

jdufresne opened the pull request:
As numpy is Python 3 only, these import statements are now unnecessary and don't alter runtime behavior.

mattip commented on tools/openblas_support.py:
```diff
@@ -1,5 +1,3 @@
-from __future__ import division, absolute_import, print_function
-import os
```
do not remove the `os` import

jdufresne committed:
MAINT: Remove unnecessary 'from __future__ import ...' statements

As numpy is Python 3 only, these import statements are now unnecessary
and don't alter runtime behavior.

jdufresne commented:
Rebased for 2020 :tada:

jarrodmillman commented:
@eric-wieser This seems like a good PR to merge next.  It only removes `__future__` imports.  I scanned all the changes and it looks good to me and it looks like it got them all.

seberg commented:
I agree, will merge after tests pass. The comment about the doc-test not being runnable confused me, but its 13 years old, so I think its just a left over from a different age.

seberg commented:
Thanks @jdufresne  and @jarrodmillman!
